CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.16.2
+* Bug - Fixed a bug where the ticker would submit empty container state change
+  transitions when a task is STOPPED [#1178](https://github.com/aws/amazon-ecs-agent/pull/1178)
```

Comment:
Do you think we need a changelog for the update of `amazon-ecs-cni-plugins`? Also please add a `.` at the end for consistency.

Reply:
i don't think we need an update for the cni plugins repo, we usually don't include dependency updates in the changelog. although i just noticed we did for `1.16.1`. if you feel we should, i can add it.

Reply:
I don't think we need to declare that we updated the dependency.  If we added a new feature, enhanced an existing feature, or fixed a bug, those are things that we can call out in the changelog.